features/step_definitions/purap/requisition.rb

```diff
@@ -1,6 +1,23 @@
-And /^I create a Requisition with required Delivery and Additional Institutional information populated$/ do
+And /^I create a Requisition with required Chart-Organization, Delivery and Additional Institutional information populated$/ do
   visit(MainPage).requisition
   on RequisitionPage do |req_page|
+    # Check Chart/Org associated with the user; if either one does not exist, use corresponding parameter default value and perform lookup
+    chart,org = req_page.chart_org_readonly.split('/')
+    (chart.nil? || chart.empty?) ? (nil) : (chart = chart.strip())
```

Comment:
Based on your comments below, it actually sounds like you should do:
```ruby
# Ensure everything is either stripped or empty
chart,org = req_page.chart_org_readonly.split('/').collect{ |x| x.nil? ? '' : x.strip } 

if org.empty? || chart.empty?
# continue from original line 9...
```

Does that seem reasonable?